new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSimpleDomainValidation.java

```diff
@@ -34,19 +39,23 @@
 
 @DisplayName("Simple validation of basic domain functions")
 @IntegrationTest
+//Every test class needs to tagged with this annotation for log collection and namespace creation.
+@ExtendWith(IntegrationTestWatcher.class)
+// The ITNamespaces numofns value is used to create the number of unique namespaces needed by the test.
+// The namespace list is available for use by all methods in the test class.
+@ITNamespaces(numofns = 1) //
```

Comment:
Wouldn't this annotation be better on the method using it?
Like:
public void testCreatingDomain(@ITNamespaces(numofns = 1) List namespaces)

Reply:
Yes we can do this. Lets see what other think.
A minor thing is, it has to be placed at every method that wants to create new namespaces.

Reply:
Done